pycheribuild/projects/cherios.py

```diff
@@ -38,10 +38,11 @@ class BuildCheriOS(CMakeProject):
     default_build_type = BuildType.DEBUG
     repository = GitRepository("https://github.com/CTSRD-CHERI/cherios.git", default_branch="master")
     _default_install_dir_fn = ComputedDefaultValue(
-        function=lambda config, cls: config.output_root / ("cherios" + config.mips_cheri_bits_str),
+        function=lambda config, cls: config.output_root / ("cherios" + cls._xtarget.build_suffix(config,
+                                                                                                 include_os=False)),
         as_string="<OUTPUT_ROOT>/cherios128")
```

Comment:
Outdated

Reply:
What is preferred?

Reply:
Something like `cherios-{mips64,riscv64}` which I believe are the two values you'll get (though check that...)

Reply:
Though as you're changing this line, switch it to $OUTPUT_ROOT, the <...> syntax is mostly phased out

Reply:
Ah wait, are we talking about the outdated cherios128? cherios-mips and cherios-riscv64 are the two values I otherwise get.

Reply:
Yeah. And I see, that's because CHERIOS_MIPS_PURECAP never got updated to have its suffix be mips64 rather than mips like all the others did. Should probably just fix that.

Reply:
Updated the suffix